src/site/stages/build/process-cms-exports/transform.js

```diff
@@ -1,10 +1,23 @@
+/* eslint-disable import/no-dynamic-require */
+
+const fs = require('fs');
+const path = require('path');
+
 const { mapKeys, camelCase } = require('lodash');
 const { getContentModelType } = require('./helpers');
-const pageTransform = require('./transformers/page');
 
-const transformers = {
-  'node-page': pageTransform,
-};
+// Dynamically read in all the transformers
+// They must be named after the content model type (E.g. node-page.js)
+const transformersDir = path.join(__dirname, 'transformers');
+const transformers = fs
+  .readdirSync(transformersDir)
+  .filter(name => name.endsWith('.js'))
+  .reduce((t, fileName) => {
+    const contentModelType = fileName.slice(0, -3); // Take of the '.js'
+    // eslint-disable-next-line no-param-reassign
+    t[contentModelType] = require(path.join(transformersDir, fileName));
+    return t;
+  }, {});
```

Comment:
I think this is a clever thing and I know you did it in one of your earlier PRs.  Now that we've done it more than once, would there be value in pulling it out into some `importFromDir(dirName)` function?

Reply:
:thinking: Yeah, we might could refactor it, but for now, they do handle things a little differently.